apps/file-service/src/docs/swagger.ts

```diff
@@ -0,0 +1,40 @@
+import { INestApplication } from '@nestjs/common';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import * as basicAuth from 'express-basic-auth';
+import { SWAGGER_CONFIG } from './swagger.config';
+
+/**
+ * Creates an OpenAPI document for an application, via swagger.
+ * @param app the nestjs application
+ * @returns the OpenAPI document
+ */
+const SWAGGER_ENVS = ['local', 'development', 'production'];
+
+export function createDocument(app: INestApplication) {
+  const builder = new DocumentBuilder()
+    .setTitle(SWAGGER_CONFIG.title)
+    .addBearerAuth(
+      { type: 'http', scheme: 'bearer', bearerFormat: 'JWT' },
+      'authorization',
+    )
+    .setDescription(SWAGGER_CONFIG.description)
+    .setVersion(SWAGGER_CONFIG.version);
+  for (const tag of SWAGGER_CONFIG.tags) {
+    builder.addTag(tag);
+  }
+  const options = builder.build();
+  const { username, password }: any = { username: 'hello', password: 'hello' };
```

Comment:
"I think we can manage these variable in the env sa,e as others can be picked up from runtime configuration"